LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -212,6 +213,15 @@ public struct CorrectionRangeOverridesEditor: View {
             dismiss()
         }
     }
+
+    private func accessibilityIdentifier(for preset: CorrectionRangeOverrides.Preset) -> String {
+        switch preset {
+        case .preMeal:
+            return "pre-meal"
```

Comment:
so will have as `pre-meal_picker` and `workout_picker`

Reply:
Yep!